tables/ru-ru-g1.ctb

```diff
@@ -0,0 +1,664 @@
+# liblouis: Russian literary (grade 1) text translation table
+#
+#-index-name: Russian, grade 1
+#-display-name: Russian contracted braille
+#
+#+locale: ru
+#+type: literary
+#+dots: 6
+#+contraction: partial
+#+grade:1
+#+direction: forward
```

Comment:
Doesn't it make sense to try to make the table work in both directions?

Reply:
The problem is that this system forbids use capsletter indicator as the dots 45 are used for contraction. I don't think it makes sense now because only lowercase letters can be translated back.
Besides, the problem with Latin alphabet. In Russian uncontracted braille I added back translation using `context` rules for each letter.
But I'll try to study this question when you'll present a new feature to add foreign alphabets.